app/ui/browser-modern/views/browser/page/index.jsx

```diff
@@ -130,6 +179,9 @@ class Page extends Component {
     return (
       <div id={`browser-page-${this.props.pageId}`}
         className={`browser-page ${PAGE_STYLE}`}>
+        <ErrorPage
+          hidden={this.props.pageState.load !== PageState.STATES.FAILED}
+          {...this.props.pageState.toJS()} />
```

Comment:
Ugh, please let's not uselessly convert immutable js objects to js objects for no reason. Update the ErrorPage to handle immutable objects instead.